api/routes.py

```diff
@@ -152,4 +166,18 @@ def put_to_toolhub(tool, data):
   header = {"Authorization": f'Bearer {flask.session["token"]["access_token"]}'}
   response = requests.put(url, data=data, headers=header)
   r = response.status_code
-  return r
\ No newline at end of file
+  return r
+
+user = Blueprint("user", __name__, description="Get information about the currently logged-in user.")
+
+@user.route("/api/user")
+class CurrentUser(MethodView):
+  @tasks.response(200, UserSchema)
+  def get(self):
+    """Get the username of currently logged-in user."""
+    response = get_current_user()
+    if type(response) == str:
+      username = {"username": response}
+      return jsonify(username)
```

Comment:
I missed this one before: flask-smorest automagically "jsonifies" responses for you, so you can just lazily return python types everywhere :)

Reply:
Ah, I think I've done that in a couple of places.  Can tweak.